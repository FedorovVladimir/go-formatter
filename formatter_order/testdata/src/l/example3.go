package main

// v2 - 2 // want "formatter_order"

var v2 = "2"

// c2 - 2 // want "formatter_order"

const c2 = "2"
