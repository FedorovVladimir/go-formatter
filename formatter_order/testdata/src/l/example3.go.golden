package main

// c2 - 2 // want "formatter_order"

const c2 = "2"

// v2 - 2 // want "formatter_order"

var v2 = "2"
